velox/functions/prestosql/benchmarks/JsonExprBenchmark.cpp

```diff
@@ -39,6 +39,14 @@ class JsonBenchmark : public velox::functions::test::FunctionBenchmarkBase {
         {"folly_is_json_scalar"});
     registerFunction<SIMDIsJsonScalarFunction, bool, Json>(
         {"simd_is_json_scalar"});
+    registerFunction<JsonArrayContainsFunction, bool, Json, bool>(
+        {"folly_json_array_contains"});
+    registerFunction<SIMDJsonArrayContainsFunction, bool, Json, bool>(
+        {"simd_json_array_contains"});
+    registerFunction<JsonArrayLengthFunction, int64_t, Json>(
+        {"folly_json_array_length"});
+    registerFunction<SIMDJsonArrayLengthFunction, int64_t, Json>(
```

Comment:
Thanks, Can we add a benchmark to compare string performance ? That would be a very common use case.